channel.go

```diff
@@ -281,6 +281,26 @@ func (ch *Channel) RemoveMembers(userIDs []string, message *Message) error {
 	return nil
 }
 
+type RoleAssignment struct {
+	// UserID is the ID of the user to assign the role to.
+	UserID string `json:"user_id"`
+
+	// ChannelRole is the role to assign to the user.
+	ChannelRole string `json:"channel_role"`
+}
+
+// AssignRoles assigns roles to members with given IDs.
+func (ch *Channel) AssignRole(assignments []*RoleAssignment, msg *Message) error {
+	data := map[string]interface{}{"assign_roles": assignments}
```

Comment:
can we check `if assignements == nil` first?

Reply:
✅